app/jobs/process_merge_time_metrics_job.rb

```diff
@@ -0,0 +1,7 @@
+class ProcessMergeTimeMetricsJob < ApplicationJob
```

Comment:
this will be a cron?

Reply:
Ohhh yes! I will put it in the scheduler

Reply:
should we assign them different names? something like `queue_as :merge_time` so it can be readable in sidekiq,  what do you think?